reward_index/migrations/11_add_mobile_promotion_reward_type.sql

```diff
@@ -0,0 +1 @@
+ALTER TYPE reward_type ADD VALUE 'mobile_promotion';
```

Comment:
I don't think we need a new rewad_type, I think this should just be `mobile_subscriber` or `mobile_gateway` depending on key of promotion reward.

Reply:
This promotion type has been removed in favor of using the already existing `MobileSubscriber` and `MobileGateway` members.